apps/web/components/signin/CredentialsForm.tsx

```diff
@@ -44,6 +46,14 @@ function SignIn() {
     resolver: zodResolver(signInSchema),
   });
 
+  if (clientConfig.auth.disablePasswordAuth) {
+    return (
```

Comment:
If we do this, api key signin errors won't get rendered. Maybe you should inline the error handling here as well?